README.md

```diff
@@ -39,6 +41,22 @@ from tree_sitter import Language, Parser
 PY_LANGUAGE = Language(tspython.language())
 ```
 
+#### Wasm support
+
+If you enable the `wasm` extra, then tree-sitter will be able to use wasmtime to load languages compiled to wasm and parse with them. Example:
+
+```python
+from pathlib import Path
+from wasmtime import Engine
+from tree_sitter import Language, Parser
+
+engine = Engine()
+wasm_bytes = Path("my_language.wasm").read_bytes()
```

Comment:
Perhaps you could explain how to compile or where to download `my_language.wasm`?

Reply:
https://tree-sitter.github.io/tree-sitter/creating-parsers#command-build